default.nix

```diff
@@ -44,7 +44,7 @@ let iosSupport = system == "x86_64-darwin";
 
     bindHaskellOverlays = self: super: {
       haskell = super.haskell // {
-        overlays = super.overlays or {} // import ./haskell-overlays {
+        overlays = super.haskell.overlays or {} // import ./haskell-overlays {
```

Comment:
This should have been `super.haskell.overlays` all along; it only once worked because there was no top level `overlays` attribute in Nixpkgs. But now there is.